### DISCUSSION TITLE
Rustdesk calls admin.rustdesk.com, which does not exist.

### DISCUSSION THREAD

**Discussion by Freekers** [rustdesk/rustdesk#1352] [Q&A] [answered]
Rustdesk calls 'admin.rustdesk.com', which is a non-existent subdomain of Rustdesk.com

https://github.com/rustdesk/rustdesk/blob/c1906914a0f8e3089c8cec3e4eb644af91302bc7/flutter/lib/pages/connection_page.dart#L301

https://github.com/rustdesk/rustdesk/blob/5baed21fceae9d42e1f5f5c4e7ad645eb5c35721/flutter/lib/pages/settings_page.dart#L381

https://github.com/rustdesk/rustdesk/blob/dc4757fe94006156752040a8fc0b70ccea9fd330/src/common.rs#L580

**Comment by Freekers** [comment]
Why was this converted into a discussion? There's nothing to discuss; a domain is being called that does not exist. Seems like a bug to me.

**Comment by crhope** [comment]
This may be a new feature in the future, waiting to be developed.

**Comment by BT-nolongeractive-bye** [comment]
I would hazard a guess based on the surrounding code it has to do with login feature that is yet to be implemented, also a little more concerning is what is found in this discussion https://github.com/rustdesk/rustdesk/discussions/1440

**Comment by robinhugy** [comment]
Still not working :(

**Comment by KleinDevDE** [comment]
Windows x64 MSI
I still cannot login because admin.rustdesk.com is not valid.

This exists in v1.2.4, but also in v1.2.6
![image](https://github.com/rustdesk/rustdesk/assets/13904220/7c730027-72cc-41ed-985a-e8da011ebe42)
![image](https://github.com/rustdesk/rustdesk/assets/13904220/d3050563-d92c-4c03-ae11-4ca4e5921bc5)

**Comment by rustdesk** [comment, marked as answer]
login is used for rustdesk server pro for the time being.

https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/

**Reply by KleinDevDE** [reply]
Good to know

Then admin.rustdesk.com is some kind of placeholder, right?
The current error message only leads to confusion and bug reports/discussions like this.

How about doing one of the following instead?
- If no configuration has been loaded, the login window shows a small message with a link to the documentation.
- If no configuration has been loaded, the login window (and login menu entry) is not displayed at all
- Enter the server hostname in the login form
